cloudinit/configure-falcon-sensor.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+"""Set the customer ID for the CrowdStrike Falcon sensor.
+
+This file is a template.  It must be processed by Terraform.
+"""
+
+from __future__ import annotations
+
+# Standard Python Libraries
+# Bandit triggers B404 here, but we're only using subprocess.run() and
+# doing so safely.  For more details on B404 see here:
+# https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess
+import subprocess  # nosec
+import sys
+from typing import Any
+
+# Third-Party Libraries
+import boto3
+
+# Inputs from Terraform
```

Comment:
Based on what I've seen, I know we've been sticking to alphabetic sorting. Can you please organize this alphabetically? 

```suggestion
FALCON_CUSTOMER_ID_KEY: str = "${falcon_customer_id_key}"
FALCON_SENSOR_INSTALL_PATH: str = "${falcon_sensor_install_path}"
FALCON_TAGS_KEY: str = "${falcon_tags_key}"
SSM_READ_ROLE_ARN: str = "${ssm_read_role_arn}"
SSM_REGION: str = "${ssm_region}"
```

Reply:
Please see commit 3186a29.